src/pages/Home.js

```diff
@@ -10,6 +10,8 @@ import footer_decal from '../assets/cmdf_footerdecal.svg'
 const Decal = styled.img`
   pointer-events: none;
   object-fit: cover;
+  overflow: hidden;
+  z-index: -1;
 `
 
 const TopDecal = styled(Decal)`
```

Comment:
can we change the 275px in left: and width: on lines 19 and 21 to 265px so that one lilypad isn't floating in space hehe